node/service/src/lib.rs

```diff
@@ -640,7 +647,7 @@ where
 			spawner,
 			parachain_consensus,
 			import_queue,
-			collator_key,
+			collator_key: collator_key.expect("Expected collator key"), // TODO: be more graceful / better error
```

Comment:
This could use review. It happens during process startup, so I'm not immediately worried about it.

Reply:
We could maybe map it to this error https://github.com/paritytech/substrate/blob/56dc6d1a3d9ae166f47368f9006ff4fe5457d889/client/service/src/error.rs#L64

Reply:
I fixed this but you can change the string of the error if you want